### PR TITLE
Delete category

### PR DESCRIPTION
**Pull request by mvn-trangle2-dn**
## Description
delete category
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

mysql> explain delete from `categories` where `parent_id` = 18 \G;
*************************** 1. row ***************************
           id: 1
  select_type: DELETE
        table: categories
   partitions: NULL
         type: ALL
possible_keys: NULL
          key: NULL
      key_len: NULL
          ref: NULL
         rows: 13
     filtered: 100.00
        Extra: Using where
1 row in set (0.00 sec)

### CODE REVIEW

File: app/Http/Controllers/Admin/CategoryController.php [hunk 1]
```diff
@@ -27,6 +29,80 @@ public function index()
...
+    }
+
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param int $id category's id
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
```
Comment (at-huenguyen) [inline app/Http/Controllers/Admin/CategoryController.php]: hàm xóa category, cách thực hiện sẽ như nào em?

**Comment by at-phuongvu** [comment]
LGTM

File: app/Models/Category.php [hunk 2]
```diff
@@ -17,6 +19,8 @@ class Category extends Model
         'name', 'parent_id', 'level'
     ];
     
+    protected $dates = ['deleted_at'];
```
Comment (at-vietpham) [inline app/Models/Category.php]: Vì sao lại phải thêm `deleted_at` vào `dates`

File: public/js/deleteCategory.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+function deleteCategory(e, id) {
+  e.preventDefault();
+  msg = "Do you want to delete Category Id= " + id;
```
Comment (at-vietpham) [inline public/js/deleteCategory.js]: Message phải được gọi từ file lang

**Review by ghost — CHANGES_REQUESTED** [review]

File: app/Models/Category.php (hunk 2 above)
Comment (at-vietpham) [reply app/Models/Category.php]: Không cần cho vào vẫn dùng soft delete được mà?

**Comment by at-vietpham** [comment]
LGTM 💴

**Pull request merged**